src/task/task.service.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  BadRequestException,
+  HttpCode,
+  HttpStatus,
+  Injectable,
+} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Task } from 'src/entities/task.entity';
+import { formatDate } from 'src/helpers/date-and-time.helper';
+import { Repository } from 'typeorm';
+import { UpdateTaskDto } from './dto/update-task.dto';
+import { GetUser } from 'src/decorator/get-user.decorator';
+import { User } from 'src/entities/user.entity';
+
+@Injectable()
+export class TaskService {
+  constructor(
+    @InjectRepository(Task)
+    private taskRepository: Repository<Task>,
+  ) {}
+
+  async getDefaultTaskList(
+    workspaceId: number,
+    userId: number,
```

Comment:
userId should be probably changed to user, which should be injected in the controller with GetUser decorator